doc/dev/developer_guide/essentials.rst

```diff
@@ -159,6 +159,26 @@ register an account. Registering gives you a unique IRC identity and allows you
 to access channels where unregistered users have been locked out for technical
 reasons.
 
+To register your account on Ceph's IRC channels, type the following into the text field in your IRC client::
+
+  /msg NickServ REGISTER irc_username username@foo.com
```

Comment:
irc_username here should be password

Reply:
I'm not sure about that, Josh. I just checked my irc history, and I ran the command as I have it, not as you're suggesting. It worked.

I know that kind of leaves us at an impasse, and I don't mean to shut the conversation down, so: what now?

Reply:
linking to the oftc docs sounds good, and future-proof